site/docs/plugins/conversations.md

```diff
@@ -1256,6 +1256,44 @@ Internally, when several wait calls are reached at the same time, the conversati
 As soon as the next update arrives, it will then replay the conversation builder function once for each encountered wait call until one of them accepts the update.
 Only if none of the pending wait calls accepts the update, the update will be dropped.
 
+## Waiting for One of Many Wait Calls
+
+You can wait for multiple of wait calls simultaneously using the [Promise.race](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/race) method.
```

Comment:
This is incorrect. Using `Promise.race` is a bad idea and will behave unexpectedly. It will resolve with the first update, but then it will swallow all other updates until all wait calls have resolved. These updates are lost forever (simply due to how `Promise.race` works) so it is never useful.